packages/neuron-wallet/src/services/hardware/ledger.ts

```diff
@@ -110,8 +110,8 @@ export default class Ledger extends Hardware {
       .pipe(
         // searching for 2 seconds
         takeUntil(timer(2000)),
-        filter<DescriptorEvent<Descriptor>>(e => e.type === 'add'),
-        scan<DescriptorEvent<Descriptor>, DeviceInfo[]>((acc, e) => {
+        filter<DescriptorEvent<any>>(e => e.type === 'add'),
```

Comment:
Are there no types replacing `Descriptor`?

Reply:
Yes,`Descriptor` had been deprecated in [@ledgerhq/hw-transport
(6.20.0)](https://www.npmjs.com/package/@ledgerhq/hw-transport#listen).